lib/spack/docs/packaging_guide.rst

```diff
@@ -2344,6 +2344,37 @@ you set ``parallel`` to ``False`` at the package level, then each call
 to ``make()`` will be sequential by default, but packagers can call
 ``make(parallel=True)`` to override it.
 
+Note that the `--jobs` option works out of the box for all standard
+build systems. If you are using a non-standard build system instead, you
+can use the variable `make_jobs` to extract the number of jobs specified
+by the `--jobs` option:
+
+.. code-block:: python
+   :emphasize-lines: 15, 20
+   :linenos:
+ 
+   class Xios(Package):
+      ...
+
+      def install(self, spec, prefix):
+         ...
+
+         options = [
+            "--full",
```

Comment:
I would only list the relevant arguments for this example. Using a real package is nice as it gives users a place to go for further details.

Reply:
I removed all the arguments except the one passing `str(make_jobs)`